rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -85,6 +85,86 @@
           }
         }
       }
+    },
+    "holdings_type": {
+      "type": "string",
+      "title": "Type of the holdings",
+      "enum": [
+        "monograph",
+        "serial"
+      ],
+      "default": "serial"
+    },
+    "patterns": {
+      "title": "Captions and patterns information",
+      "type": "array",
+      "items": {
+        "title": "Captions and patterns",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "name": {
+            "type": "string",
+            "title": "The name of the pattern."
+          },
+          "levels": {
+            "title": "Enumeration and chronology levels",
+            "type": "array",
+            "items": {
+              "title": "Enumeration and chronology levels",
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "title": "The name of the level."
+                },
+                "starting_value": {
+                  "type": "integer",
+                  "title": "The starting value of current level."
+                },
+                "show": {
+                  "type": "boolean",
+                  "title": "To show this level at issue display or not.",
+                  "default": true
+                },
+                "next_value": {
+                  "type": "integer",
+                  "title": "The next value of current level."
+                },
+                "completion_value": {
+                  "type": "integer",
+                  "title": "Units per next higher level."
+                },
+                "mapping_values": {
+                  "title": "Mapping values for chronology",
+                  "type": "array",
+                  "uniqueItems": true,
+                  "minItems": 1,
+                  "items": {
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
     }
+  },
+  "if": {
+    "properties": {
+      "holdings_type": {
+        "const": "periodic"
```

Comment:
Perhaps it's correct but I don't see term "periodic" in enum of `holding_type`.

Reply:
fixed